db/migrate/20150826165942_add_subscription_facet.rb

```diff
@@ -15,8 +15,8 @@ def change
                     :name => "katello_subscription_facets_host_id", :column => "host_id"
 
     create_table "katello_subscription_facet_activation_keys" do |t|
-      t.references :subscription_facet, :null => false, :index => true
-      t.references :activation_key, :null => false, :index => true
+      t.references :subscription_facet, :null => false, :index => { :name => "katello_subscription_facet_activation_keys_sf" }
+      t.references :activation_key, :null => false, :index => { :name => "katello_subscription_facet_activation_keys_ak" }
```

Comment:
I think the index would function the same way in the database and only the name would be different. ```AR docs say``` it would generate something that looks like

```CREATE UNIQUE INDEX by_branch_party ON accounts(branch_id, party_id)```